examples/leveldb/main.cpp

```diff
@@ -0,0 +1,6 @@
+#include <leveldb/db.h>
+
+int main()
+{
+    return 0;
```

Comment:
Can you add anything that uses leveldb to check linking is working correctly?

Reply:
Lol TBH I never do that but I guess you are correct.

Reply:
Can we do this as a follow-up in next pull request? Tests almost finished (3 OSX job left) and I'm ready to merge this one. The rest looks good.

Reply:
Yes.